### PR TITLE
Temporary - Wake up computers during atdp hours

### PR DESCRIPTION
**Pull request by NotRyan**
Requested by cooperc. 

Wake up lab computers between 8:15 AM and 4:30 PM on Mondays, Wednesdays, and Fridays for ATDP. This is not meant to be permanent.

### CODE REVIEW

File: staff/lab/lab-wakeup [hunk 1]
```diff
@@ -34,7 +47,7 @@ def main(argv=None):
     args = parser.parse_args(argv)
     force = args.force or args.host
 
-    if force or read_hours_listing().is_open():
+    if force or read_hours_listing().is_open() or is_atdp_hours:
```
Comment (kpengboy) [inline staff/lab/lab-wakeup]: You need to call the `is_atdp_hours` function here

File: staff/lab/lab-wakeup [hunk 2]
```diff
@@ -21,7 +23,18 @@ def wake_hosts(filter, quiet):
         ['wakeonlan'] + [host['macAddress'][0] for host in hosts_by_filter(filter)],
         stdout=subprocess.DEVNULL if quiet else sys.stdout,
     )
+    
+def is_atdp_hours(when=None):
+        """Return whether the given time is during atdp hours.
```
Comment (kpengboy) [inline staff/lab/lab-wakeup]: You should be using 4 space indents here. Consider installing pre-commit hooks?

File: staff/lab/lab-wakeup [hunk 3]
```diff
@@ -21,7 +23,18 @@ def wake_hosts(filter, quiet):
...
 
+        If not provided, when defaults to now.
+        """
+        days = [0, 2, 4] #Monday, Wednesday, Friday
```
Comment (kpengboy) [inline staff/lab/lab-wakeup]: Nit: suggested comment style: two spaces between the code and the comment, space after the comment character

**Comment by cg505** [comment]
same comments as @kpengboy, otherwise lgtm

thank you!

**Pull request merged**